static/about.html

```diff
@@ -12,6 +12,30 @@
   <link href="css/bootstrap-4.2.1.css" rel="stylesheet">
   <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
   <script src="https://kit.fontawesome.com/3aa9d240b2.js" crossorigin="anonymous"></script>
+  <style>
```

Comment:
what is this code used for? I don't know that we need it

Reply:
oh whoops. I used that i when i was messing around with some stuff. I deleted it now.